Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1273,9 +1271,51 @@ def instantiateVariableFont(
         },
     )
 
+    if updateFontNames:
+        # Set Regular/Italic/Bold/Bold Italic bits as appropriate, after the
+        # name table has been updated.
+        setRibbiBits(varfont)
+
     return varfont
 
 
+def setRibbiBits(font):
+    """Set the `head.macStyle` and `OS/2.fsSelection` style bits
+    appropriately."""
+
+    english_ribbi_style = font["name"].getName(2, 3, 1, 0x409)
```

Comment:
Nit. We have a NameID enum in https://github.com/fonttools/fonttools/blob/main/Lib/fontTools/varLib/instancer/names.py so we could do: 

```PYTHON
font["name"].getName(NameID.SUBFAMILY_NAME, 3, 1, 0x409)
```